api/graphql/mutations/sequencing_group.py

```diff
@@ -36,3 +59,30 @@ async def add_comment(
             content=content,
         )
         return GraphQLComment.from_internal(result)
+
+    @strawberry.mutation
+    async def update_sequencing_group(
+        self,
+        sequencing_group_id: str,
+        sequencing_group: strawberry.scalars.JSON,
```

Comment:
These inputs should follow the same pattern used elsewhere of having a input type with the id as part of it

Reply:
Should this also return the sequencing group instead of a `bool`?

Reply:
yep!